libnavigation-core/src/main/java/com/mapbox/navigation/core/telemetry/MapboxNavigationTelemetry.kt

```diff
@@ -672,6 +688,29 @@ internal object MapboxNavigationTelemetry {
         sendEvent(freeDriveEvent)
     }
 
+    private fun createCustomEvent(
+        payload: String,
+        customEventType: String,
+        customEventVersion: String,
+        phoneState: PhoneState,
+        onEventUpdated: ((NavigationCustomEvent) -> Unit)?
+    ) {
+        log("customEventType: $customEventType")
+        val customEvent =
+            NavigationCustomEvent().apply {
+                this.payload = payload.plus(", userId = ${phoneState.userId}")
+                this.type = customEventType
+                this.driverMode = "freeDrive"
```

Comment:
let's put `null`, as we don't specify it

Reply:
Cannot do. It has to be either `freeDrive` or `trip` otherwise server returns an error code. cc @RingerJK

Reply:
let's put `00000000-0000-0000-0000-000000000000` as a stub